forms/collections/elements.js

```diff
@@ -26,7 +27,11 @@ define(function (require) {
 
     if (elementModel.get('forms')) {
       errorList = elementModel.get('forms').reduce(function (errList, form) {
-        return form.get('elements').reduce(addToErrorList, errList);
+        elements = form.get('elements');
+        if (!elements) {
+          return [];
```

Comment:
I did have tests in `/test/00_unit/tests/collections/elements.js/` but they dont work anymore, and i dont think they would of covered this outcome either. They were meant to be a proper unit test, where everything is excluded or mocked with known values and just one function is tested. 

im 100% certain you should be returning errList though. reduce builds upon the return value of the last iteration, by returning an empty array you're throwing the results of every previous iteration away and replacing it with nothing. 

so if the last form in a subform is deleted, the result of the reduce would be an empty array, even if 10 forms before it and 100 elements in the parent form had errors in them.